src/Elastic.Apm/Model/DbSpanCommon.cs

```diff
@@ -33,28 +33,34 @@ internal static class DefaultPorts
 				captureStackTraceOnStart);
 		}
 
-		internal void EndSpan(Span span, IDbCommand dbCommand, Outcome outcome, TimeSpan? duration = null)
+		internal void EndSpan(ISpan iSpan, IDbCommand dbCommand, Outcome outcome, TimeSpan? duration = null)
```

Comment:
minor nitpick: keep this as `span` and rename the `if (iSpan is Span span)` to something else e.g. `capturedSpan`